src/Sulu/Bundle/AutomationBundle/Serializer/TaskSerializerSubscriber.php

```diff
@@ -63,9 +82,14 @@ public function onTaskSerialize(ObjectEvent $event)
         }
 
         $handler = $this->handlerFactory->create($object->getHandlerClass());
-
         if ($handler instanceof AutomationTaskHandlerInterface) {
             $event->getVisitor()->addData('taskName', $handler->getConfiguration()->getTitle());
         }
+
+        $task = $this->taskRepository->findByUuid($object->getTaskId());
+        $executions = $this->taskExecutionRepository->findByTask($task);
```

Comment:
i have introduced the interface of the library than it can retrieve the executions by uuid.